portal/src/main/resources/content/examples_su2c.markdown

```diff
@@ -1,7 +1,7 @@
 <p>
 - <a href="index.do?Action=Submit&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_mutations&data_priority=0&case_set_id=ov_tcga_3way_complete&Z_SCORE_THRESHOLD=2.0&cancer_study_list=ov_tcga&cancer_study_id=ov_tcga&RPPA_SCORE_THRESHOLD=2.0&gene_list=BRCA1+BRCA2&tab_index=tab_visualize&gene_set_choice=user-defined-list&">BRCA1/BRCA2 mutations in Ovarian Cancer</a>
 <br><br>
-- <a href="index.do?genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_mutations&Z_SCORE_THRESHOLD=1.0&gene_list=PTEN%3A+Homdel+MUT%3B&tab_index=tab_visualize&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=ov_tcga_gistic&Action=Submit&data_priority=0&case_set_id=ov_tcga_3way_complete&cancer_study_list=ov_tcga&cancer_study_id=ov_tcga&RPPA_SCORE_THRESHOLD=2.0&gene_set_choice=user-defined-list&">PTEN Alterations in Ovarian Cancer</a>
-<br><br>
-- <a href="index.do?tab_index=tab_visualize&cancer_study_list=brca_tcga&cancer_study_id=brca_tcga&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=brca_tcga_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=brca_tcga_gistic&Z_SCORE_THRESHOLD=1.0&case_set_id=brca_tcga_all&case_ids=&gene_list=ERBB2%3A+amp%3B%0D%0AIGF1R%3A+amp%3B%0D%0APIK3CA%3A+amp%3B+%0D%0APTEN%3A+homdel%3B%0D%0A%0D%0A%0D%0A%0D%0A&gene_set_choice=user-defined-list&Action=Submit">Pi3K Pathway Alterations in Breast Cancer</a>
```

Comment:
The old links still work if applying frontend PR [3578](https://github.com/cBioPortal/cbioportal-frontend/pull/3578) to them with the JS bookmark, but these changes are just for consistency